documentation/library_guide/oneDPL_2022.0_changes.rst

```diff
@@ -0,0 +1,40 @@
+oneDPL 2022.0 Changes
+#####################
+
+This page lists breaking changes in oneDPL 2022.0 from oneDPL 2021.
+
+Support of C++11 and C++14 has been discontinued. 
```

Comment:
Support for C++11 and C++14 has been discontinued.

Reply:
Applied the suggestion.